drivers/hmis/app/graphql/schema.graphql

```diff
@@ -10057,6 +10111,15 @@ input ServiceTypeFilterOptions {
   searchTerm: String
 }
 
+"""
+Create service type input
+"""
+input ServiceTypeInput {
+  customServiceCategoryId: ID!
```

Comment:
We have been keeping the `custom` out of the graphql schema to date. So to be in keeping with that, this should probably be just `serviceCategoryId`.